parlai/core/torch_generator_agent.py

```diff
@@ -875,15 +877,26 @@ def eval_step(self, batch):
             beam_preds_scores, beams = self._generate(
                 batch, self.beam_size, maxlen, prefix_tokens=prefix_tokens
             )
-            preds, scores = zip(*beam_preds_scores)
+            preds, _, _, _= zip(*beam_preds_scores)
             self._add_generation_metrics(batch, preds)
 
             # bsz x beamsize
             beam_texts: List[List[Tuple[str, float]]] = []
+            beam_generated_text_token_info: List[List[Tuple[str, float, int]]] = []
+
             for beam in beams:
                 beam_texts.append([])
-                for tokens, score in beam.get_rescored_finished():
+                if self.output_token_losses:
```

Comment:
how is this getting set

Reply:
should it just be always true?

Reply:
This is a flag that's already being used in `main` to determine whether to show token level probabilities for labels